cpp/src/Ice/ACM.cpp

```diff
@@ -129,9 +129,7 @@ IceInternal::FactoryACMMonitor::add(const ConnectionIPtr& connection)
     if (_connections.empty())
     {
         _connections.insert(connection);
-        _instance->timer()->scheduleRepeated(
-            shared_from_this(),
-            chrono::duration_cast<chrono::nanoseconds>(_config.timeout) / 2);
+        _instance->timer()->scheduleRepeated(shared_from_this(), _config.timeout / 2);
```

Comment:
scheduleRepeated is a template - no need to convert the unit of the timeout.

Reply:
Would be good to have a comment.